Amazon.SQS/src/Extensions.SQS/Extensions.SQS.csproj

```diff
@@ -20,8 +20,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SQS" Version="3.5.1.8" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
+    <PackageReference Include="AWSSDK.SQS" Version="3.5.1.21" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.25" />
```

Comment:
why are we updating the version of webjobs SDK? Is there a dependency on 3.0.25 for this functionality? If not, I would prefer to use as low a version as possible, so it imposes least restrictions.

Reply:
Sorry, so long to respond.  I'm fine with the choice of webjobs SDK.  I have this running and receiving messages.  It's been running for months with low message volume.